app/views/components/dateInput.scala.html

```diff
@@ -21,29 +21,28 @@
 @this(govukDateInput: GovukDateInput)
 
 @(content: String,
-    field: Field,
-    hintContent: Option[String] = None,
-    smallText: Boolean = false,
-    largeText: Boolean = true,
-    labelClass: Option[String] = None,
-    pageHeading: Boolean = true
+        field: Field,
+        hintContent: Option[Content] = None,
+        smallText: Boolean = false,
+        largeText: Boolean = true,
+        labelClass: Option[String] = None,
```

Comment:
not sure if UI wise there will be ever situation when labelClass will be None? maybe not worth to model this field like Optional?